rskj-core/src/test/java/co/rsk/peg/federation/NonStandardErpFederationsTest.java

```diff
@@ -61,7 +55,7 @@ class NonStandardErpFederationsTest {
 
     @BeforeEach
     void setup() {
-        BridgeConstants bridgeConstants = BridgeMainNetConstants.getInstance();
+        FederationConstants federationConstants = FederationMainNetConstants.getInstance();
```

Comment:
would it make sense to call them `federationMainnetConstants`?

Reply:
updated